astropy_helpers/tests/test_openmp_helpers.py

```diff
@@ -7,6 +8,7 @@
 
 IS_TRAVIS_LINUX = os.environ.get('TRAVIS_OS_NAME', None) == 'linux'
 IS_APPVEYOR = os.environ.get('APPVEYOR', None) == 'True'
+PY3_LT_35 = sys.version_info[0] == 3 and sys.version_info[1] < 5
```

Comment:
Why not do  if sys.version_info[:2] < (3, 5), does it work for py2.7?

Reply:
Yes it does work for Python 2.7. It's an MVCS version thingy.

Reply:
oh, ok then, sorry for the noise